pkg/operator/ceph/version/version.go

```diff
@@ -310,7 +310,7 @@ func ValidateCephVersionsBetweenLocalAndExternalClusters(localVersion, externalV
 
 	// Local version must never be higher than the external one
 	if IsSuperior(localVersion, externalVersion) {
-		return errors.Errorf("local cluster ceph version is higher %q than the external cluster %q, this must never happen", externalVersion.String(), localVersion.String())
```

Comment:
The check is for `IsSuperior()`, which means we are only trying to validate if the local version is higher than the external ceph version, which isn't supported. Why isn't this message accurate? The suggested message isn't accurate since we do not require the versions to match.

Reply:
I think the message is correct